al-sufi-arabic/description.md

```diff
@@ -0,0 +1,160 @@
+# Al-Şūfī Arabic
+
+## Introduction
+This sky culture reflects the indigenous star names, constellations and asterisms identified by the prominent astronomer: Al-Şūfī in his **Book of the Stars** (Arabic:كتاب الكواكب), written in the 10th century. The book maps the Arab names to the respective stars of the 48 classical Greek constellations in Ptolemy's Almagest.
```

Comment:
Please use this style for emphasis _Book of the Stars_ instead of **Book of the Stars**

Reply:
I mean, using underscore _ instead of **

Reply:
Done.